vowpalwabbit/parse_args.cc

```diff
@@ -1003,6 +1006,13 @@ void parse_reductions(vw& all)
   all.reduction_stack.push_back(bs_setup);
 
   all.l = setup_base(all);
+
+  if (all.probabilities)
+  { if (!all.vm.count("oaa") && !all.vm.count("csoaa_ldf"))
+      THROW("--probabilities can only be used with --oaa=N or --csoaa_ldf=mc");
+    if (!all.vm.count("loss_function") || all.vm["loss_function"].as<string>() != "logistic" )
```

Comment:
I would need to repeat the test both in oaa.cc and csoaa.cc. Should I do it?
I felt the test (just two lines of code) fits here where I am testing `all.probabilities && (!oaa || !csoaa_ldf)` (and this test obviously cannot be moved to oaa.cc/csoaa.cc).